### DISCUSSION TITLE
How can I get validation loss in mmpose?

### DISCUSSION THREAD

**Discussion by seon-creator** [open-mmlab/mmpose#2967] [Q&A]
Hello, I checking tools of mmpose.
After finishing model train, I checked vis_data/scalars.json.
In json there is two kinds of line, loss, AP.
Like this

```
{"lr": 4.754709418837672e-05, "data_time": 0.7217564284801483, "loss": 0.003005887003382668, "loss_kpt": 0.003005887003382668, "acc_pose": 0.4137931034482758, "time": 1.143358051776886, "epoch": 1, "iter": 48, "memory": 12758, "step": 48}
{"coco/AP": 0.0, "coco/AP .5": 0.0, "coco/AP .75": 0.0, "coco/AP (M)": -1.0, "coco/AP (L)": 0.0, "coco/AR": 0.0, "coco/AR .5": 0.0, "coco/AR .75": 0.0, "coco/AR (M)": -1.0, "coco/AR (L)": 0.0, "data_time": 0.3225211302439372, "time": 0.4114294449488322, "step": 1}
```

And I want to know is that loss value is train loss or validation loss?

**Comment by Ben-Louis** [comment]
These values are training losses. MMPose does not report validation loss by default.
